### PR TITLE
Lab7

### PR DESCRIPTION
**Pull request by afonyaa**

### CODE REVIEW

**Comment by none-word** [comment]
Good job!

**Comment by Maxkoz777** [comment]
Well done!

File: monitoring/promtail.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+server:
+  http_listen_port: 9080
+  http_listen_address: 0.0.0.0
+  grpc_listen_port: 0
```
Comment (AKurmazov) [inline monitoring/promtail.yml:4]: nit: You can remove this redundant statement

**Comment by AKurmazov** [comment]
Well done! 👍

**Pull request closed without merging**